reco_utils/recommender/deeprec/models/base_model.py

```diff
@@ -437,10 +432,9 @@ def fit(self, train_file, valid_file, test_file=None):
 
             if self.hparams.save_model:
                 if epoch % self.hparams.save_epoch == 0:
-                    save_path_str = join(self.hparams.MODEL_DIR, "epoch_" + str(epoch))
                     checkpoint_path = self.saver.save(
                         sess=train_sess,
-                        save_path = save_path_str,
+                        save_path=self.hparams.MODEL_DIR + "epoch_" + str(epoch),
```

Comment:
this was recently changed by @elogicaadith on this PR https://github.com/microsoft/recommenders/pull/1011, he said it was not working. Can you check?

Reply:
It's caused by the slash '/' char in path str. I haven't tried in in AML.  But since it is a recently changed, I will update it. Thanks!